yowsup/common/http/warequest.py

```diff
@@ -165,3 +173,39 @@ def sendRequest(host, port, path, headers, params, reqType="GET"):
 
         response = conn.getresponse()
         return response
+
+    def sendRequestWithProxy(host,port,path,headers,params,proxy):
+        import pycurl
+        import json
+        from io import BytesIO
+        logger.info("SENDING PROXY REQUEST WITH %s"%proxy.getHost())
+        bytes_buffer = BytesIO()
+        c = pycurl.Curl()
+        c.setopt(pycurl.URL, WARequest.build_get_url(host,path,params))
+        c.setopt(pycurl.PROXY,proxy.getHost())
+        c.setopt(pycurl.PROXYPORT,proxy.getPort())
+        if proxy.getUserName() is not None:
+            c.setopt(pycurl.PROXYUSERPWD, "%s:%s" % (proxy.getUser(), proxy.getPassword()))
+        c.setopt(pycurl.PORT, port)
+        c.setopt(pycurl.HTTPHEADER,WARequest.build_headers(headers))
+        c.setopt(pycurl.WRITEDATA, bytes_buffer)
+        c.perform()
+        c.close()
+        data = bytes_buffer.getvalue().decode('utf-8')
+        return json.loads(data)
+
+    @staticmethod
+    def build_get_url(host, path, params):
+        params = urlencode(params)
+        url = 'https://'+host+path+"?"+params 
+        print(url)
```

Comment:
Is it possible to remove debug code? I know this PR is already merged, but better without debug prints.